browser/ui/webui/brave_webui_source.cc

```diff
@@ -23,6 +24,16 @@
 #include "ui/base/l10n/l10n_util.h"
 #include "ui/base/webui/resource_path.h"
 
+#if defined(TOOLKIT_VIEWS)
```

Comment:
You can delete this check all in this PR.
There is no view toolkit related code.